From: Andriy Grytsenko <andrej@rep.kiev.ua>
Date: Thu, 14 Sep 2017 14:25:15 +0300
Subject: hide_properties

---
 src/users/callbacks.c | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/src/users/callbacks.c b/src/users/callbacks.c
index 08455f4..4054c62 100644
--- a/src/users/callbacks.c
+++ b/src/users/callbacks.c
@@ -63,7 +63,7 @@ on_table_selection_changed (GtkTreeSelection *selection, gpointer data)
 }
 
 static void
-do_popup_menu (GtkTreeView *treeview, GdkEventButton *event)
+do_popup_menu (GtkTreeView *treeview, GdkEventButton *event, gint table)
 {
 	GtkTreeSelection *selection;
 	GtkUIManager *ui_manager;
@@ -91,6 +91,9 @@ do_popup_menu (GtkTreeView *treeview, GdkEventButton *event)
 	gtk_widget_set_sensitive (gtk_ui_manager_get_widget (ui_manager, "/MainMenu/Delete"),
 				  cont > 0);
 
+	if (table == TABLE_USERS)
+		gtk_widget_hide (gtk_ui_manager_get_widget (ui_manager, "/MainMenu/Properties"));
+
 	gtk_menu_popup (GTK_MENU (popup), NULL, NULL, NULL, NULL,
 			button, event_time);
 }
@@ -120,7 +123,7 @@ on_table_button_press (GtkTreeView *treeview, GdkEventButton *event, gpointer da
 
 			gtk_tree_path_free (path);
 
-			do_popup_menu (treeview, event);
+			do_popup_menu (treeview, event, table);
 		}
 
 		return TRUE;
@@ -136,7 +139,7 @@ on_table_button_press (GtkTreeView *treeview, GdkEventButton *event, gpointer da
 gboolean
 on_table_popup_menu (GtkTreeView *treeview, GtkWidget *popup)
 {
-	do_popup_menu (treeview, NULL);
+	do_popup_menu (treeview, NULL, -1);
 	return TRUE;
 }
 
@@ -158,6 +161,8 @@ on_popup_settings_activate (GtkAction *action, gpointer data)
 
 	if (table == TABLE_GROUPS)
 		on_group_settings_clicked (NULL, NULL);
+	else if (table == TABLE_USERS)
+		return; // do nothing
 }
 
 void
